SalesforceSDK/Universal/Analytics/SalesforceAnalyticsManager.cs

```diff
@@ -0,0 +1,347 @@
+                }
+                if (Instances != null)
+                {
+                    var manager = Instances[uniqueId];
+                    if (manager != null)
+                    {
+                        await manager.analyticsManager.ResetAsync();
+                    }
+                    Instances.Remove(uniqueId);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Returns an instance of event store manager.
+        /// </summary>
+        /// <returns>Event store manager</returns>
+        public EventStoreManager GetEventStoreManager()
+        {
+            return eventStoreManager;
+        }
+
+        /// <summary>
+        /// Returns an instance of analytics manager.
+        /// </summary>
+        /// <returns>Analytics manager.</returns>
+        public AnalyticsManager GetAnalyticsManager()
+        {
+            return analyticsManager;
+        }
+
+        /// <summary>
+        /// Disables or enables logging of events. If logging is disabled, no events
+        /// will be stored. However, publishing of events is still possible.
+        /// </summary>
+        /// <param name="enabled"></param>
+        public void DisableEnableLogging(bool enabled)
+        {
+            eventStoreManager.DisableEnableLogging(enabled);
+        }
+
+        /// <summary>
+        /// Returns whether logging is enabled or disabled
+        /// </summary>
+        /// <returns>True - if logging is enabled, False - otherwise</returns>
+        public bool IsLoggingEnabled()
+        {
+            return eventStoreManager.IsLoggingEnabled();
+        }
+
+        /// <summary>
+        /// Publishes all stored events to all registered network endpoints after
+        /// applying the required event format transforms. Stored events will be
+        /// deleted if publishing was successful for all registered endpoints.
+        /// This method should NOT be called from the main thread.
+        /// </summary>
+        /// <returns></returns>
+        public async Task PublishAllEventsAsync()
+        {
+            var events = await eventStoreManager.FetchAllEventsAsync();
+            await PublishEventsAsync(events);
+        }
+
+        /// <summary>
+        /// Publishes a list of events to all registered network endpoints after
+        /// applying the required event format transforms. Stored events will be
+        /// deleted if publishing was successful for all registered endpoints.
+        /// This method should NOT be called from the main thread.
+        /// </summary>
+        /// <param name="events"></param>
+        /// <returns></returns>
+        public async Task PublishEventsAsync(List<InstrumentationEvent> events)
+        {
+            if (events == null || events.Count == 0)
+            {
+                return;
+            }
+            var eventIds = new List<string>();
+            var success = true;
+            var remoteKeySet = remotes.Keys;
```

Comment:
@kchitalia Where are you adding the default transformer and publisher, a.k.a. `AILTNTransform` and `AILTNPublisher`, to the list of remotes?